src/test/java/hw5/pages/DifferentElementsPage.java

```diff
@@ -0,0 +1,40 @@
+package hw5.pages;
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.FindBy;
+import org.openqa.selenium.support.ui.Select;
+
+import java.util.List;
+
+public class DifferentElementsPage extends AbstractPage {
+
+    public DifferentElementsPage(WebDriver driver) {
```

Comment:
What is the proper place in the class for constructor according to **Java Code Convention**?